ui/apps/dashboard/src/components/Onboarding/utils.ts

```diff
@@ -11,3 +11,7 @@ export function getNextStepName(currentStepName: OnboardingSteps): OnboardingSte
 
   return nextStep ? nextStep.name : undefined;
 }
+
+export const ONBOARDING_VERCEL_NEXT_URL = encodeURIComponent(
+  'https://app.inngest.com/env/production/onboarding/deploy-app'
+);
```

Comment:
Ideally we could use `NEXT_PUBLIC_APP_URL` env var here so this is dynamic for local or prod